OnbordingTemplates/DeployDcr/PerfAndMapDcr/DeployDcrParameters.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "WorkspaceResourceId": {
+            "value": "/subscriptions/<SubscriptionId>/resourceGroups/<ResourceGroup>/providers/Microsoft.OperationalInsights/workspaces/<WorkspaceName>"
+        },
+        "WorkspaceLocation": {
+            "value": "<WorkspaceLocation>"
+        },
+        "userGivenDcrName": {
+            "type": "String",
+            "metadata": {
+              "displayName": "Name of the Data Collection Rule(DCR)",
+              "description": "This is the name of the AMA-VMI Data Collection Rule(DCR)"
+            },
+            "defaultValue": "ama-vmi-default-dcr"

```

Comment:
May be add the perf and map indication in the DCR name. No need of default keyword.

Reply:
sure, will make the, change, default name is line with the Ux offering where a default name is provided